app/MMB/src/scenes/flightServices/FlightServicesMenuGroup.js

```diff
@@ -106,13 +106,16 @@ export class FlightServicesMenuGroup extends React.Component<Props, State> {
   getOrderedAndNotOrderedServices = () => {
```

Comment:
Please comment this behavior even in the code - the logic is getting fairly complicated.

Reply:
See if my comments makes sense 🙈